client/my-sites/plans/ecommerce-trial/style.scss

```diff
@@ -0,0 +1,191 @@
+	}
+
+	.e-commerce-trial-plans__trial-card {
+		display: flex;
+		padding: 40px;
+		background-color: rgba(var(--color-neutral-0-rgb), 0.8);
+		box-shadow: none;
+
+		@media (max-width: $break-mobile) {
+			padding: 24px;
+			margin: 0 20px;
+		}
+
+		.e-commerce-trial-plans__trial-card-content {
+			flex: 1;
+			margin-top: 10px;
+
+			.e-commerce-trial-plans__card-title {
+				font-size: $font-title-medium;
+				font-weight: 400;
+				color: var(--studio-black);
+				margin: 0;
+
+				@media (max-width: $break-mobile) {
+					font-size: $font-body-small;
+				}
+			}
+
+			.e-commerce-trial-plans__card-subtitle {
+				font-size: $font-body;
+				font-weight: 400;
+				margin: 15px 0;
+				color: var(--color-neutral-40);
+
+				@media (max-width: $break-mobile) {
+					font-size: $font-body-extra-small;
+					color: var(--color-neutral-80);
+				}
+			}
+		}
+
+		.e-commerce-trial-plans__chart-wrapper {
+			@media (max-width: $break-mobile) {
+				display: none;
+			}
+
+			margin-left: 50px;
+			padding: 0 20px;
+			text-align: center;
+
+			.e-commerce-trial-plans__chart-label {
+				color: var(--color-neutral);
+			}
+		}
+	}
+
+	.e-commerce-trial-plans__interval-toggle-wrapper {
+		display: flex;
+		justify-content: center;
+		// Margins from designs
+		margin: 48px 0 32px 0;
+		padding: 0 20px;
+
+		.price-toggle {
+			width: 240px;
+
+			@media (max-width: $break-mobile) {
+				width: 100%;
+			}
+		}
+	}
+
+	.e-commerce-trial-plans__price-card {
+		display: flex;
+		flex-wrap: wrap;
+		border-radius: 2px 2px 0 0;
+		padding: 32px;
+		margin: 0;
+
+		@media (max-width: $break-mobile) {
+			flex-direction: column;
+			margin: 0 20px;
+			padding: 32px 20px;
+		}
+
+		.e-commerce-trial-plans__price-card-text {
+			flex: 1;
+			display: flex;
+			flex-direction: column;
+
+			.e-commerce-trial-plans__price-card-title {
+				font-size: $font-title-medium;
+			}
+
+			.e-commerce-trial-plans__price-card-subtitle {
+				flex: 1;
+				font-size: $font-body-small;
+				color: var(--color-neutral-80);
+			}
+		}
+
+		.e-commerce-trial-plans__price-card-cta-wrapper {
+			width: 100%;
+
+			.e-commerce-trial-plans__price-card-cta {
+				width: 135px;
+
+				@media (max-width: $break-mobile) {
+					width: 100%;
+					margin-top: 25px;
+					line-height: 30px;
+				}
+			}
+		}
+
+		.e-commerce-trial-plans__price-card-conditions {
+			display: flex;
+			flex-direction: column;
+			text-align: right;
+
+			@media (max-width: $break-mobile) {
+				text-align: left;
+			}
+
+			.e-commerce-trial-plans__price-card-value {
+				font-size: $font-title-large;
+				font-weight: 500;
+
+				@media (max-width: $break-mobile) {
+					font-size: $font-headline-small;
+					margin-top: 30px;
+					font-weight: 400;
+				}
+
+				.e-commerce-trial-plans__price-card-value-symbol {
+					@media (max-width: $break-mobile) {
+						// Make a <sup>-like style
+						/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+						font-size: 70%;
```

Comment:
Can we use `0.7em` instead? I am not sure if that's possible, but it would be better to be explicit so we don't need to ignore the style rule.

Reply:
`em` is also not allowed, but I just put the specific size for this instance (`$font-title-medium`)